### PR TITLE
typos and recommendation

### PR DESCRIPTION
**Pull request by baartch**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 ### Properties
 
-- [#app](API.md##app)
```
Comment (baartch) [inline API.md:11]: I assume this file is autogenerated. Am I right? So this error will probably popup again.

File: subgraph/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Currently there are Subgraphs deployed on the [xdai](https://thegraph.com/explor
 
 ## Queries
 
-Below there are a few ways to show how to query the Subgraph for data.  The queries show most of the information that is queryable, but there are many other filtering options that can be used, just check out the [querying api](https://thegraph.com/docs/graphql-api). These queries can be used locally or in The Graph Explorer playground.
```
Comment (baartch) [inline subgraph/README.md:9]: This is just a recommendation. I hope I got this right.

Comment (PJColombo) [reply subgraph/README.md:9]: It's great!

**Review by PJColombo — APPROVED** [review]
LGTM!

**Pull request merged**